kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -507,6 +507,15 @@ object KyuubiConf {
       .serverOnly
       .fallbackConf(FRONTEND_BIND_PORT)
 
+  val FRONTEND_THRIFT_CONNECTION_URL_HOST: ConfigEntry[String] =
+    buildConf("kyuubi.frontend.thrift.connection.url.host")
```

Comment:
The following configuration is used by Apache Kafka

```
advertised.host.name
advertised.port
advertised.listeners
```

I would call it `kyuubi.frontend.thrift.advertised.host`, if you want, we call it `kyuubi.frontend.advertised.host` and apply it to all frontends.

Reply:
Thanks for your advice!
`advertised.host` sounds better to me.
Fixed at 34c61d9.

I also applied the configuration property to all frontends.
But I did not introduce separate properties for each frontend at this time.
It's possible, but I think such a use case is rare.